app/interfaces/api/v1/dropdown_data.rb

```diff
@@ -9,9 +9,19 @@ class DropdownData < API::Helpers::GrapeApiHelper
         params :role_filter do
           optional :role, type: String, desc: I18n.t('api.v1.dropdown_data.params.role_filter'), values: %w[agfs lgfs]
         end
+        params :scheme_ten_role_filter do
+          optional :role,
+                   type: String,
+                   desc: I18n.t('api.v1.dropdown_data.params.role_filter'),
+                   values: %w[agfs_scheme_9 agfs_scheme_10 lgfs]
```

Comment:
As discussed i would include agfs here to and default this to behaving as agfs_scheme_9 - or vice versa - to maintain current functionality -i think this is probably needed some vendor client apps might use this filtering option to populate list items